installer/scripts/omsadmin.sh

```diff
@@ -360,7 +360,7 @@ onboard()
         clean_exit 1
     fi
 
-    if [[ ! $WORKSPACE_ID =~ ^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$ ]]; then
+    if [[ ! $WORKSPACE_ID =~ ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$ ]]; then
```

Comment:
Interesting. This script uses /bin/sh which, on RH, is the same as Bash. All I can imagine here is that the machine you found this on has an older version of Bash. Certainly RedHat would have the same issue.